app/views/devise/registrations/_edit_form.html.erb

```diff
@@ -0,0 +1,69 @@
+<%= form_for(resource, as: resource_name,
+             url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
Lets add a new line before and after it for consistency  just like you did in app/views/devise/registrations/_new_form.html.erb

Reply:
Ok fixed in other devise generated views too ;-)